### ISSUE TITLE
LSF submission problems due to "subprocess" and "bsub" submission string

### ISSUE THREAD

**Issue by kruglovIFX** [ansys/pyaedt#4287]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

In desktop.py, the line 134 causes issues:
`p = subprocess.Popen(command, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)`

Solution:
1. The "command" should first be joined as a string:
`command = " ".join(str(x) for x in command)`
2. The "subprocess" command should be passed with the following options:
`p = subprocess.Popen(command, shell = True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)`

So, to summarize, the line 134 should look like these two lines:
`command = " ".join(str(x) for x in command)`
`p = subprocess.Popen(command, shell = True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)`

Then LSF submission works.
Linux LSF, pyaedt 0.7.12

P.S. solution worked out together with Felipe Castro from Ansys

### Steps To Reproduce

Use submit_job method for any design

### Which Operating System are you using?

Linux

### Which Python version are you using?

3.10

### Installed packages

pyaedt 0.7.12
